webpages/settings/index.js

```diff
@@ -380,15 +381,19 @@ let fuse;
       window.addEventListener(
         "hashchange",
         (e) => {
-          const addonId = location.hash.replace(/^#addon-/, "");
-          const groupWithAddon = this.addonGroups.find((group) => group.addonIds.includes(addonId));
-          if (!groupWithAddon) return; //Don't run if hash is invalid
-          const addon = this.manifestsById[addonId];
-
-          groupWithAddon.expanded = true;
-          this.selectedCategory = addon?.tags.includes("easterEgg") ? "easterEgg" : "all";
-          this.clearSearch();
-          setTimeout(() => document.getElementById("addon-" + addonId)?.scrollIntoView(), 0);
+          if (location.hash.toLocaleLowerCase() === "#moresettings") {
```

Comment:
```suggestion
          if (location.hash === "#moresettings") {
```
`#addon-*` links are case-sensitive, so I think this one should be too.